lib/models/change/test.js

```diff
@@ -152,4 +159,131 @@ describe('Unit :: Change Model', () => {
       expect(employee.phone).to.equal('(123) 123-1234');
     });
   });
+
+  describe('Unit :: Tasks :: Email Changes', () => {
+    let companyUser, employees, users, permissions;
+
+    beforeEach(async () => {
+      require('../../..')();
+
+      companyUser = await makeUser();
+
+      permissions = await (new Permission({
+        key: 'update changes'
+      })).save();
+
+      employees = await Promise.all([{
+        name: { first: 'joe',
+          last:  'smith' },
+        email:       'smith@aehr.org',
+        password:    '1234',
+        company:     companyUser.company,
+        activatedOn: new Date()
+      }, {
+        name: { first: 'Bob',
+          last:  'Ross' },
+        email:       'prettytrees@aehr.org',
+        password:    '1234',
+        company:     companyUser.company,
+        activatedOn: new Date()
+      }].map(d => (new Employee(d)).save()));
+
+      users = await Promise.all([{
+        email:    'chererjeremy@gmail.com',
+        password: '124',
+        name:     {
+          first: 'Jack',
+          last:  'Frost'
+        },
+        permissions,
+        company:  companyUser.company,
+        employee: employees[ 0 ]
+      }, {
+        email:    'jeremy@aehr.org',
+        password: '124',
+        name:     {
+          first: 'Tom',
+          last:  'Johnson'
+        },
+        permissions,
+        company:  companyUser.company,
+        employee: employees[ 1 ]
+      }, {
+        email:    'chererjeremy@gmail.com',
+        password: '124',
+        name:     {
+          first: 'Dasiy',
+          last:  'Duke'
+        },
+        company: companyUser.company
+      }].map(d => (new User(d)).save()));
+    });
+
+    afterEach(async () => {
+      await mongoose.connection.dropDatabase();
+    });
+
+    it('should create two notifications each with two sets of changes', async () => {
+      await (new Change({
+        company:  companyUser.company,
+        employee: employees[0],
+        changes:  [{
+          path:     'name.first',
+          value:    'Jack',
+          employee: employees[0]
+        }]
+      })).save();
+
+      await new Promise(resolve => setTimeout(resolve, 500));
+
+      await (new Change({
+        company:  companyUser.company,
+        employee: employees[1],
+        changes:  [{
+          path:     'name.last',
+          value:    'top',
+          employee: employees[1]
+
+        }]
+      })).save();
+
+      await new Promise(resolve => setTimeout(resolve, 500));
+
+      let record = await Notification.find({
+        company:   companyUser.company,
+        debounced: true,
+        template:  'employee_change-request'
+      });
+
```

Comment:
We need to assert there are **only** `2` notifications
```suggestion
expect(record).to.be.an('array').and.have.lengthOf(2);
```

Reply:
🏁